crates/diesel_models/src/query/user/sample_data.rs

```diff
@@ -165,3 +182,30 @@ pub async fn delete_refunds(
             _ => Ok(result),
         })
 }
+
+pub async fn delete_disputes(
+    conn: &PgPooledConn,
+    merchant_id: &common_utils::id_type::MerchantId,
+) -> StorageResult<Vec<Dispute>> {
+    // delete disputes where merchant_id = merchant_id and payment_id like "test_%"
+    let query = diesel::delete(<Dispute>::table())
+        .filter(dispute_dsl::merchant_id.eq(merchant_id.to_owned()))
+        .filter(dispute_dsl::payment_id.like("test_%"));
```

Comment:
We can use like on payment_id, but I it will be better to use on dispute_id,
where `dispute_id` like `test_`, though both will produce same result. 

It will be better because we have compount index on merchant id and dispute_id and in this case, PostgreSQL can use an index on the dispute column because the prefix(test_%) is fully specified.

Also Please do the similar change this in above refund query.